classlib/Database.php

```diff
@@ -0,0 +1,272 @@
+<?php
+
+/**
+ * Database class - create single instance
+ *
+ * @package Database
+ * @subpackage
+ * @category
+ * @author Sanjeeb Rao
+ */
+
+class Database {
+    private $db = array(
+        'master' => array(
+            'host' => 'localhost',
+            'username' => 'root',
+            'password' => 'mindfire',
+            'db_name' => 'registration'
+        ),
+        'slave' => array(
+            'host' => 'localhost',
+            'username' => 'root',
+            'password' => 'mindfire',
+            'db_name' => 'registration'
+        )
+    );
+
+    private $connection = NULL;
+
+    // The single instance
+    private static $instance = NULL;
+
+    /**
+     * Creating connection.
+     *
+     * @access private constructor of Database
+     * @param  void
+     */
+    private function __construct()
+    {
+        $this->connection = new mysqli(
+            $this->db['master']['host'],
+            $this->db['master']['username'],
+            $this->db['master']['password'],
+            $this->db['master']['db_name']);
```

Comment:
Closing parenthesis can be added on new line on same indentation level as $this->connection